templates/discourse/files/config/discourse.platform.sh.conf

```diff
@@ -0,0 +1,26 @@
+env RAILS_ENV=production
+redis_host  = cache.internal
+redis_port  = 6379
+db_host     = database.internal
+db_port     = 5432
+db_name     = main
+db_username = main
+db_password = main
+smtp_address = <%= ENV['PLATFORM_SMTP_HOST']%>
+developer_emails= <%= ENV['DISCOURSE_DEVELOPER_EMAILS']%>
+# smtp authentication mechanism
+smtp_authentication = none
+# enable TLS encryption for smtp connections
+smtp_enable_start_tls = false
+serve_static_assets = true
+config.assets.compile = true
+config.assets.digest = false
+
+# you should export in the environment using the CLI or set here the specifics
+# platform vset -emaster env:DISCOURSE_DEVELOPER_EMAILS myemail@example.com
+# platform vset env:DISCOURSE_MAXMIND_LICENSE_KEY get_a_key_from_maxmind
+# platform vset -emaster env:DISCOURSE_HOSTNAME $(platform route:get --primary -P url| cut -d'/' -f3)
+
+#developer_emails=myemail@example.com
+# FIXME this should work through the environment variable and doesn't?
+hostname = <%= ENV['DISCOURSE_HOSTNAME']%>
```

Comment:
It doesn't?  Do any of the other env vars here?  I've not been able to verify.

Reply:
If it doesn't work, then why is this env var being set manually, per the instructions?  This doesn't compute.

Reply:
It seems discourse overrides the Env variables from the config file (which is really a weird take on configuration precedence. I don't want us to change any of the files in the repo if we can avoid it. So we are weirdly creating this one, which just makes sure the order of precedence is correct.